src/public/js/instructions/Instructions.jsx

```diff
@@ -0,0 +1,22 @@
+import React, {Component} from 'react';
+
+class Instructions extends Component {
```

Comment:
This was just a rename, for some reason github is treating this as an entirely new file